test/test_other.py

```diff
@@ -13374,7 +13374,7 @@ def test_split_module(self, customLoader, jspi, opt):
       self.emcc_args += ['-g', '-sJSPI_EXPORTS=say_hello']
     self.emcc_args += ['-sEXPORTED_FUNCTIONS=_malloc,_free']
     output = self.do_other_test('test_split_module.c')
-    if jspi:
+    if self.js_engines == [config.V8_ENGINE]:
       # TODO remove this when https://chromium-review.googlesource.com/c/v8/v8/+/4159854
```

Comment:
This TODO was landed - can we remove this if?

Reply:
I think we would need to do some more work there to make it work.